### PR TITLE
prevent blank message inputs

### PR DESCRIPTION
**Pull request by leeannachen**

### CODE REVIEW

File: proj/src/main/webapp/chat.js [hunk 1]
```diff
@@ -428,23 +428,29 @@ function initRef(dbRefObject) {
 
 function pushChatMessage() {
     const messageInput = document.getElementById('message-input');
-
-    var message = {
+    // prevent blank messages
+    if (messageInput.value.trim().length != 0){
+        var message = {
         content : messageInput.value,
```
Comment (aokeke) [inline proj/src/main/webapp/chat.js:433]: I'm not sure if it is what you want, but maybe you should include the `.trim()` here also when you're storing it. It means you can't have messages prefixed or suffixed with white space which can either be good or bad. Just for you to think about.

File: proj/src/main/webapp/chat.js [hunk 2]
```diff
@@ -428,23 +428,29 @@ function initRef(dbRefObject) {
...
         timestamp : new Date().getTime(),
         senderDisplay : firebase.auth().currentUser.displayName,
         senderUID : firebase.auth().currentUser.uid
```
Comment (aokeke) [inline proj/src/main/webapp/chat.js:436]: nit: can you indent this block.

**Review by aokeke — APPROVED** [review]
This looks like it has some changes that are might also be present in your other PR. After you merge that one, pull master into this branch to clean up the extra diffs. You might see some merge conflicts.

File: proj/src/main/webapp/chat.js (hunk 1 above)
Comment (leeannachen) [reply proj/src/main/webapp/chat.js:433]: this maybe very niche but i think trimming spaces would mess up those ASCII pic/art people occasionally send in chat

File: proj/src/main/webapp/chat.js (hunk 2 above)
Comment (leeannachen) [reply proj/src/main/webapp/chat.js:436]: for sure; i indented this in the branch i am currently working on

**Pull request merged**